python/pyarrow/tests/test_array.py

```diff
@@ -2643,6 +2643,15 @@ def test_concat_array_invalid_type():
         pa.concat_arrays(arr)
 
 
+def test_combine_chunks():
+    # ARROW-77363
```

Comment:
I think in general the ChunkedArray tests are in `test_table.py` I think (although this is a bit confusing, but it's also defined in table.pxi)

Reply:
For sure. Moved it